src/app/ArticleEditing/page.tsx

```diff
@@ -0,0 +1,32 @@
+"use client"
+import React, { useState } from 'react';
+import ArticleEditor from '../_components/ArticleEditor';
```

Comment:
This causes a `document is not defined` and `500` error when loading `/ArticleEditing`. Replace with following to disable SSR for this component:
```
import dynamic from 'next/dynamic';
const ArticleEditor = dynamic(() => import('@/app/_components/ArticleEditor'), {
  ssr: false,
});
```

Solution from https://github.com/vercel/next.js/discussions/15181

Reply:
Resolved. The Article Editor has been switched to using ProseMirror rather than react-quill. A lot of the imports and code has been overwritten, including the error in concern here.